qtop.py

```diff
@@ -1496,6 +1495,14 @@ def quit_program(h_start, h_stop, v_start, v_stop):  # "q", quit
 
 
 def control_movement(pressed_char_hex, h_start, h_stop, v_start, v_stop):
+    """
+    Basic vi-like movement is implemented for the -w switch (linux watch-like behaviour for qtop).
+    h, j, k, l for left, down, up, right, respectively.
+    Both g/G and Shift+j/k go to top/bottom of the matrices
+    0 and $ go to far left/right of the matrix, respectively.
+    r resets the screen to its initial position (if you've drifted away from the vieweable part of a matrix).
+    q quits qtop.
+    """
```

Comment:
Nice PR, lots of little things coming in; I hadn't realised the use of key "q", nice to have some help :-P